### DISCUSSION TITLE
FIXED: Uncaught exception - Invalid field access : entities

### DISCUSSION THREAD

**Discussion by EldieSoft** [HaxeFlixel/flixel#3297] [General]
Hello all,

I've been moving from Java to Haxe recently, and I am really enjoying it. I've been working through the Tutorial on haxeflixel.com, and I've run into problems with the loadEntities() method. When I place my player using the constructor (e.g. player = new Player(144, 144)) it works, but when I try to do map.loadEntities(placeEntities, "entities"), I get this error:

Called from flixel.addons.editors.ogmo.FlxOgmo3Loader::loadEntities line 174
Uncaught exception - Invalid field access : entities

I have no idea how to fix this, and the Haxe language is generally under-documented online. Can anyone figure out what's going on?

Here is my create method in PlayState.hx:

```haxe
override public function create()
	{

		map = new FlxOgmo3Loader(AssetPaths.Fungeon__ogmo, AssetPaths.room_001__json);
		walls = map.loadTilemap(AssetPaths.tiles__png, "walls");
		walls.follow();
		walls.setTileProperties(1, NONE);
		walls.setTileProperties(2, ANY);
		add(walls);

		coins = new FlxTypedGroup<Coin>();
		add(coins);


		player = new Player();
		
		map.loadEntities(placeEntities, "entities"); //it really doesnt like this very much. this is where the problem is.
		//we'll see about fixing that later.
		add(player);

		FlxG.camera.follow(player, TOPDOWN, 1);

		super.create();
	}
```

**Comment by Geokureli** [comment]
For sharing code can you please either relevant snippets via code tags:
\`\`\`haxe
// Code here
\`\`\`

Becomes
```haxe
// Code here
```

Or share full files via [gist](https://gist.github.com/) 

As for this issue, ill try to recreate it myself

**Comment by EldieSoft** [comment]
FIXED:

After a few days of tinkering, I discovered that my Entity Layer was called "Entity Layer" instead of "entities."
